RELEASE_NOTES.md

```diff
@@ -1,5 +1,79 @@
 This file holds "in progress" release notes for the current release under development and is intended for consumption by the Chef Documentation team. Please see <https://docs.chef.io/release_notes.html> for the official Chef release notes.
 
+# Chef Infra Client 15.7
+
+## Updated Resources
+
+### archive_file
+
+The `archive_file` will now only change ownership on files and directories that were part of the archive itself. This is important so you don't change permissions on important high level directories such as /etc or /bin just by extracting a file into those directories. Thanks for this fix [@bobchaos](https://github.com/bobchaos/)
+
+### cron and cron_d
+
+The `cron` and `cron_d` resources now include a `timeout` property, which allows you to configure actions to perform when a job times out. This property accepts a hash of timeout configuration options:
+
+- `preserve-status`: true/false with a default of false
+- `foreground`: true/false with a default of false
+- `kill-after`: Integer for the timeout in seconds
+- `signal`: String or Integer to send to the process such as `HUP`
+
+### launchd
+
+The launchd resource has been updated to properly capitalize `HardResourceLimits`. Thanks for this fix [@rb2k](https://github.com/rb2k/).
+
+### sudo
+
+The `sudo` resource no longer fails on the second Chef Infra Client run when using a `Cmnd_Alias`. Thanks for reporting this issue [@Rudikza](https://github.com/Rudikza)
+
+### user
+
+The `user` resource on aix no longer forces the user to change the password after Chef Infra Client modifies the password. Thanks for this fix [@Triodes](https://github.com/Triodes)
+
+The `user` resource on macOS 10.15 has received several important fixes to improve logging and prevent failures.
+
+### windows_task
+
+The `windows_task` resource is now idempotent when a system is joined to a domain and the job runs under a local user account.
+
+### x509_certificate
+
+The `x509_certificate` resource now includes a new `renew_before_expiry` property that allows you to auto renew certicates with a number of days before they expire. Thanks [@julienhuon](https://github.com/julienhuon/) for this improvement.
```

Comment:
In this first sentence, should "with" be used here? Check me on this for clarity, but I think it would be fine without "with".

Reply:
Fixed